modules/appeals_api/spec/workers/appeals_api/weeky_error_report_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe AppealsApi::WeeklyErrorReport, type: :job do
+  describe '#perform' do
+    it 'sends mail' do
+      Flipper.enable(:decision_review_weekly_error_report_enabled)
+      expect(AppealsApi::WeeklyErrorReportMailer).to receive(:build)
+        .once
+        .and_return(double.tap do |mailer|
+          expect(mailer).to receive(:deliver_now).once
+        end)
+
+      described_class.new.perform
+    end
+
+    it 'does not send report email if flipper disabled' do
+      Flipper.disable(:decision_review_weekly_error_report_enabled)
+      expect(AppealsApi::WeeklyErrorReportMailer).not_to receive(:build)
```

Comment:
I don't think this assertion is testing anything. You'll never receive `build` if you haven't even tried to run the report job, right?

Reply:
Good catch, thanks.  

I _intended_ to call perform twice, once with perform actually making the underlying WeeklyErrorReportMailer.build() call, and once with the report flipper flag disabled and perform early-outing and not calling WeeklyErrorReportMailer.build()

Added line to call perform again in the disabled test with the flag off.